code/datums/looping_sounds/machinery_sounds.dm

```diff
@@ -42,4 +42,4 @@
 	mid_sounds = list('sound/machines/microwave/microwave-mid1.ogg'=10, 'sound/machines/microwave/microwave-mid2.ogg'=1)
 	mid_length = 10
 	end_sound = 'sound/machines/microwave/microwave-end.ogg'
-	volume = 90
\ No newline at end of file
+	volume = 90
```

Comment:
I don't know how this happened but alright

Reply:
if you're just looking at this and not the full changes, this isn't actually an addition it's a newline at end of file change